proxy/http2/Http2Stream.cc

```diff
@@ -520,35 +577,47 @@ Http2Stream::initiating_close()
     bool sent_write_complete = false;
     if (_sm) {
       // Push out any last IO events
-      if (write_vio.cont) {
+      // First look for active write or read
+      if (write_vio.cont && write_vio.nbytes > 0 && write_vio.ndone == write_vio.nbytes &&
```

Comment:
This can possibly sent out a duplicate VC_EVENT_WRITE_COMPLETE/EOS to the HttpSM.
The HttpSM code is not setup to handle this, for example: if this happens after the tunnel_handler_post, we crash on the assert in tunnel_handler (expect event==EVENT_TUNNEL_DONE). 
I have a fix running which has booleans for the read write vios, I'll add a pull request later